### ISSUE TITLE
Outdated function calls in the README

### ISSUE THREAD

**Issue by matsuli** [arduino-libraries/ArduinoIoTCloud#269]
Hey, confused beginner here...

In the README section regarding the watchdog timer shouldn't the function calls be 
```
ArduinoCloud.update()
ArduinoCloud.begin(...) 
```

instead of

```
ArduinoIoTCloud.update()
ArduinoIoTCloud.begin(...)
```

**Comment by aentinger** [comment]
Good catch, thank you! It should be fixed with #270, can you take a look?

**Comment by matsuli** [comment]
Looks correct!
